jslack-api-model/src/main/java/com/github/seratch/jslack/api/model/User.java

```diff
@@ -49,6 +49,8 @@
     @Data
     public static class Profile {
 
+        private String guestChannels;
```

Comment:
I haven’t checked the actual response yet but is it a string value? Not an array?

Reply:
Just made sure it and I found it's a string value having an array inside.
```
        "guest_channels": "[\"C88888888\",\"C88888888\",\"C88888888\"]",
        "guest_invited_by": "U88888888",
```